front/app/api/content_builder/keys.ts

```diff
@@ -0,0 +1,17 @@
+import { QueryKeys } from 'utils/cl-react-query/types';
+
+const baseKey = { type: 'content_builder' };
```

Comment:
Is the baseKey correct? In the types file, I see:  `type: 'content_builder_layout';` so just want to double-check.